site/content/en/docs/Guides/Client SDKs/unity.md

```diff
@@ -226,3 +226,27 @@ The properties for the Unity Agones SDK can be found in the Inspector.
   - Whether the server sends a health ping to the Agones sidecar. (default: `true`)
 - Log Enabled
   - Debug Logging Enabled. Debug logging for development of this Plugin. (default: `false`)
+
+## Contributing
```

Comment:
Ooh thanks for this!

This is currently our development guide: https://github.com/googleforgames/agones/blob/main/build/README.md (which admittedly, is probably at a point where it should be split into multiple files). - we generally keep Agones development docs out of our user documentation (see https://github.com/googleforgames/agones?tab=readme-ov-file#development-and-contribution) 

So I'm of two minds, and would love thoughts. Do we:

1. Add a **SDK Development** section to `build/README.md` (which would be useful in general)
2. Add a new file specifically for "Developing, Testing and Building SDKs" and link to it under https://github.com/googleforgames/agones?tab=readme-ov-file#development-and-contribution

I'm actually leaning towards No. 2 - what do other people think? @igooch you've also been doing SDK work for a bit, WDYT? (Not for this PR, but this could be a home for explaining how to generate grpc code, write unit tests, conformance tests, etc)

Reply:
I do agree that the build/README.md is getting to be rather unwieldy, so I'm also leaning towards option No. 2.

Reply:
Absolutely, I'm happy to share my experiences on the subject to the project/agones community! It's not entirely intuitive to set up (at least I don't think so). Glad to possibly help anyone develop the agones unity sdk.

I'll complete these steps soon
[ ] Move agones unity client sdk contributing guide into new `.md` file under `build/`
[ ] Link to new `.md` file to "Development and Contribution" section of `README.md`